utils/mock-data-loader/re-insert-mocks.js

```diff
@@ -17,22 +17,34 @@ const { logger, LOGGER_COLOURS } = require('./helpers/logger.helper');
 
 const init = async () => {
   logger.info('REINSERTING MOCKS', { colour: LOGGER_COLOURS.bright });
-
-  const portalToken = await createAndLogInAsInitialUser();
-
-  await cleanAllTables(portalToken);
-  await insertMocks(portalToken);
-  await cleanAllTablesGef(portalToken);
-  await insertMocksGef(portalToken);
-
-  await cleanAllTablesTfm();
-
-  const tfmToken = await createAndLogInAsInitialTfmUser();
-  await insertMocksTfm(tfmToken);
-
-  await deleteInitialTFMUser(tfmToken);
-  await deleteInitialUser(portalToken);
-
+  try {
+    const portalToken = await createAndLogInAsInitialUser();
+
+    await cleanAllTablesPortal(portalToken);
+    await cleanAllTablesGef(portalToken);
+    await cleanAllTablesTfm();
+
+    await insertMocks(portalToken);
+    await insertMocksGef(portalToken);
+
+    const tfmToken = await createAndLogInAsInitialTfmUser();
+    await insertMocksTfm(tfmToken);
+
+    await deleteInitialTFMUser(tfmToken);
+    await deleteInitialUser(portalToken);
+  } catch (error) {
+    logger.warn('An error occurred, attempting to clean all tables');
```

Comment:
```suggestion
    logger.warn('An error occurred, attempting to clean all tables %o', error);
```

Reply:
I'll answer why I'm not doing this below :)